integration-to-maven-project-converter/src/main/java/com/redhat/ipaas/project/converter/DefaultIntegrationToProjectConverter.java

```diff
@@ -128,11 +129,11 @@ public DefaultIntegrationToProjectConverter(ConnectorCatalog connectorCatalog) {
                 }
 
                 try {
-                    ObjectNode stepNode = (ObjectNode) OBJECT_MAPPER.readTree(step.getConfiguredProperties());
-                    stepNode.set("kind", new TextNode(step.getStepKind()));
-                    if (step.getStepKind().equals(StepKinds.LOG)) {
-                        flow.addStep(OBJECT_MAPPER.readValue(stepNode.toString(), Log.class));
-                    }
+                    HashMap<String, Object> stepJSON = new HashMap<>(step.getConfiguredProperties());
+                    stepJSON.put("kind", step.getStepKind());
+                    String json = Json.mapper().writeValueAsString(stepJSON);
```

Comment:
If OBJECT_MAPPER is not used anymore, maybe change everytin to Json.mapper(). At least there should be no mixed usage (and eventually the constant OBJECT_MAPPER removed)